source/PlayerInfo.cpp

```diff
@@ -841,14 +841,30 @@ void PlayerInfo::AddShip(const shared_ptr<Ship> &ship)
 
 
 
-// Buy a ship of the given model, and give it the given name.
-void PlayerInfo::BuyShip(const Ship *model, const string &name)
+// Charge the player the cost of a ship and give it the specified name:
```

Comment:
This comment doesn't need to change.
```suggestion
// Buy a ship of the given model, and give it the given name.
```